Dockerfile.deb-packager

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch-slim
+ARG PROJ_VERSION="6.1.0"
+RUN apt update && apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev libssl-dev clang
+
+# Download and unzip PROJ
+WORKDIR /tmp
+RUN wget https://github.com/OSGeo/PROJ/releases/download/${PROJ_VERSION}/proj-${PROJ_VERSION}.tar.gz
+RUN tar -xzvf proj-${PROJ_VERSION}.tar.gz
+
+# Compile PROJ and install in /usr
+WORKDIR proj-${PROJ_VERSION}
+RUN ./configure --prefix=/usr
```

Comment:
shouldn't we use a /opt/proj-${PROJ_VERSION} prefix, so that this package doesn't conflict with a regular proj package ? I can imagine that people might want to still use their GDAL, QGIS, etc stack from the packaging system. And have newer PROJ alongside

Reply:
First of all, note that all of this build happen in a Docker container so it won't pollute the host.

In the hypothesis that they install it through the Debian package manager, I think they should expect only one version (unless `apt-get` allow to manage multiple versions).

I could see a rational to do what you propose if the package was manually installed with `dpkg -i proj_6.1.0_amd64.deb`. But ideally, this file would be hosted on some Debian package repository and it would be installed through `apt-get` (which means only one version installed on the machine as far as I know).